cmd/draino/draino.go

```diff
@@ -67,11 +68,9 @@ func main() {
 		leaderElectionRetryPeriod   = app.Flag("leader-election-retry-period", "Leader election retry period.").Default(DefaultLeaderElectionRetryPeriod.String()).Duration()
 		leaderElectionTokenName     = app.Flag("leader-election-token-name", "Leader election token name.").Default(kubernetes.Component).String()
 
-		skipDrain             = app.Flag("skip-drain", "Whether to skip draining nodes after cordoning.").Default("false").Bool()
-		evictDaemonSetPods    = app.Flag("evict-daemonset-pods", "Evict pods that were created by an extant DaemonSet.").Bool()
-		evictStatefulSetPods  = app.Flag("evict-statefulset-pods", "Evict pods that were created by an extant StatefulSet.").Bool()
```

Comment:
I've asked folks to preserve these flags in the past. It looks like it could be done here - we've got one in the case of `--node-label` already.

Given we no longer publish draino:latest, I'm less slightly less concerned about breaking compatibility at the moment. 

This does largely make me feel like a changelog, semantic versioning scheme, and deprecation policy for the project would be helpful.

Reply:
If we can remove the flags that would help to keep a simple code base.
If you prefer I can reintroduce them, they can exist next to the new flags, but that may confuse the users. If we keep them we may add a deprecation notice.
If ok for you I prefer to remove the old flags.